### ISSUE TITLE
code for visualization in the object detection tutorial

### ISSUE THREAD

**Issue by isalirezag** [pytorch/vision#1610]
At the end of the [object detection tutorial ](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html#torchvision-object-detection-finetuning-tutorial) it  visualizes the masks.
can you please provide the code for that task? or guide how to do it?

**Comment by fmassa** [comment]
Hi,

The code I used for the masks uses OpenCV (which torchvision doesn't use as a dependency), and that's one of the reasons why I didn't share that code yet.

Here are the scripts: https://gist.github.com/fmassa/d802c684512bcc99f54b714f10c96872

I used something like the following to visualize

```python
model = torchvision.models.detection.keypointrcnn_resnet50_fpn(pretrained=True)
model.cuda()
model.eval();

# img is a PIL Image
result, output, top_pred = predict(img, model)
plt.imshow(Image.fromarray(result[:, :, [2, 1, 0]]))
```

Let me know if you have questions. I think it makes sense to add those functions somewhere in torchvision, but I'm not yet sure where

**Comment by mattans** [comment]
@fmassa , since you used a pretrained model, didn't you need to normalize your PIL image before?

**Comment by fmassa** [comment]
Hi @mattans 

Normalization for the detection models happen inside the model, see https://github.com/pytorch/vision/blob/9b804659e1accdafc5cf3872ccbcdd1330844dbe/torchvision/models/detection/transform.py#L104

**Comment by mattans** [comment]
Great, thanks.
Linking to https://github.com/pytorch/vision/issues/2397#issuecomment-655141156 as it's related.
